### PR TITLE
Hotfix/more person identifiers

### PR DESCRIPTION
**Pull request by VirginiaDooley**
Ref https://github.com/DemocracyClub/yournextrepresentative/issues/2138, https://github.com/DemocracyClub/WhoCanIVoteFor/issues/716

At the moment, we are just serving profile urls rather than offering any reformatting with usernames linked to profiles. Happy to come back to that if urgent now, but thought I would get this done for now. 

Screenshot 2024-03-20 at 6 42 54 PM

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1206913268288723

### CODE REVIEW

File: ynr/apps/candidatebot/tests/test_candidatebot.py [hunk 1]
```diff
@@ -44,15 +45,21 @@ def test_edit_fields(self):
...
             "twitter_username": "",
             "mastodon_username": "",
             "wikidata_id": "",
             "wikipedia_url": "",
             "youtube_profile": "",
         }
         got = person.versions
+        import pdb
+
+        pdb.set_trace()
```
Comment (symroe) [inline ynr/apps/candidatebot/tests/test_candidatebot.py:62]: :lady_beetle:

File: ynr/apps/people/forms/forms.py [hunk 2]
```diff
@@ -58,8 +58,12 @@ class PersonIdentifierForm(forms.ModelForm):
         "facebook_personal_url",
         "party_ppc_page_url",
         "linkedin_url",
-        "facebook_page_url",
+        "`facebook_page_url",
```
Comment (symroe) [inline ynr/apps/people/forms/forms.py:61]: Typo?

Comment (VirginiaDooley) [reply ynr/apps/people/forms/forms.py]: messy! have fixed both. 🙏

**Review by pmk01 — APPROVED** [review]

**Comment by pmk01** [comment]
Whoops, just realised it's styled 'Bluesky'

**Review by symroe — APPROVED** [review]

**Pull request merged**